torchtext/csrc/vocab_factory.h

```diff
@@ -1,56 +1,24 @@
 #include <pybind11/pybind11.h>
+#include <pybind11/stl.h>
```

Comment:
@joecummings I would actually move this header to the corresponding `.cpp` file since we don't do any type casting to STL containers in the header. Sorry for the extra work here.